Automattic/wp-calypso
# i18n: Update textdomain for Event Countdown, Timeline and Blog Posts block

#### Changes proposed in this Pull Request

Updates were needed for Internationalization of Event Countdown, Timeline and Blog Posts blocks. This PR therefore moves the source for the Event Countdown and Timeline blocks into the FSE plugin and makes the updates to i18n, including the "full-site-editing" text-domain and add to missing strings.

The goals for FSE plugin changed so now it makes sense just to keep the source and blocks together here.

The two blocks were integrated into the build step, adding the proper package scripts. Building and testing is the same as any other FSE blocks.

#### Testing instructions

There is no intended functional change, for the two blocks, beyond including the source in the repo and the ability to translate as expected.

A full test to make sure nothing changed.

1. Create a post using the Event Countdown and Timeline block prior to apply the change.
2. Apply the change.
3. Test you can open, edit, and view the previous blocks
4. Test you can create, edit, and view new blocks

Tests on WordPress.com can be carried out with D39116-code.
